packages/languages/fi/README.md

```diff
@@ -0,0 +1,39 @@
+# @zxcvbn-ts/language-fi
+
+The Finnish dictionary and language package for zxcvbn-ts
+
+Data sources for first and last names:
+
+    https://www.avoindata.fi/data/fi/dataset/none
```

Comment:
If the name list are from this site could you directly use them please? There is already an ExcelGenerator for the nl-be language that you can use too